pkg/gui/filetree/commit_file_tree.go

```diff
@@ -2,9 +2,9 @@ package filetree
 
 import (
 	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/common"
```

Comment:
I wasn't sure if we want this dependency. It was the easiest way to implement this; without the dependency, we'd have to pass the `showRootItem` bool all the way through all the call chains that end up calling something like SetTree, and these call chains are long (starting from a class that knows the userConfig), so that would be a lot of changes. Let me know what you think @jesseduffield